### PR TITLE
Added `qute.trace.server` setting to `package.json`

### LINKED ISSUE #425: Add `qute.trace.server` as settings in package.json
Add `qute.trace.server` as settings in package.json like https://github.com/redhat-developer/vscode-xml/blob/6d27402d89734521aea77e9ca5a40792962bdd1c/package.json#L164

### PR DESCRIPTION
**Pull request by AlexXuChen**
Added `qute.trace.server` setting to `package.json`

Fixes #425

Signed-off-by: Alexander Chen alchen@redhat.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The following settings are supported:
 * `quarkus.tools.alwaysShowWelcomePage` : Determines whether to show the welcome page on extension startup. Default is `true`.
 * `quarkus.tools.starter.api` : Quarkus API base URL. Default is `https://code.quarkus.io/api`.
 * `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
+* `qute.trace.server`: Trace the communication between VS Code and the Qute language server in the Output view. Default is `off`.
```
Comment (angelozerr) [inline README.md:78]: I think we should have a `Qute support` section in the README and add this settings in the Qute support section. @fbricon any though?

Comment (AlexXuChen) [reply README.md:78]: Issue opened at https://github.com/redhat-developer/vscode-quarkus/issues/427

Comment (angelozerr) [reply README.md:78]: OK lets merge the pr. I will do the change in the documentation pr.

**Comment by angelozerr** [comment]
Thanks @AlexXuChen

**Pull request merged**